locale/ko.po

```diff
@@ -0,0 +1,2570 @@
+msgid "Crash into the HardFault_Handler.\n"
+msgstr ""
+
+#: ports/atmel-samd/common-hal/audioio/AudioOut.c
+msgid "DAC already in use"
+msgstr "DAC가 현재 사용 중입니다"
+
+#: ports/atmel-samd/common-hal/displayio/ParallelBus.c
+#: ports/nrf/common-hal/displayio/ParallelBus.c
+msgid "Data 0 pin must be byte aligned"
+msgstr ""
+
+#: shared-module/audioio/WaveFile.c
+msgid "Data chunk must follow fmt chunk"
+msgstr ""
+
+#: ports/nrf/common-hal/bleio/Broadcaster.c
+#: ports/nrf/common-hal/bleio/Peripheral.c
+msgid "Data too large for advertisement packet"
+msgstr "광고 (브로드 캐스트) 패킷에 대한 데이터가 너무 큽니다"
+
+#: ports/nrf/common-hal/bleio/Device.c
+msgid "Data too large for the advertisement packet"
+msgstr "광고 (브로드 캐스트) 패킷에 대한 데이터가 너무 큽니다"
+
+#: shared-bindings/audiobusio/PDMIn.c
+msgid "Destination capacity is smaller than destination_length."
+msgstr ""
+
+#: shared-bindings/displayio/Display.c
+msgid "Display rotation must be in 90 degree increments"
+msgstr ""
+
+#: shared-bindings/digitalio/DigitalInOut.c
+msgid "Drive mode not used when direction is input."
+msgstr ""
+
+#: ports/atmel-samd/common-hal/frequencyio/FrequencyIn.c
+#: ports/atmel-samd/common-hal/pulseio/PulseIn.c
+msgid "EXTINT channel already in use"
+msgstr ""
+
+#: extmod/modure.c
+msgid "Error in regex"
+msgstr "Regex에 오류가 있습니다."
+
+#: shared-bindings/microcontroller/Pin.c
+#: shared-bindings/neopixel_write/__init__.c shared-bindings/pulseio/PulseOut.c
+#: shared-bindings/terminalio/Terminal.c
+msgid "Expected a %q"
+msgstr "%q 이 예상되었습니다."
+
+#: shared-bindings/bleio/CharacteristicBuffer.c
+msgid "Expected a Characteristic"
+msgstr "특성(Characteristic)이 예상되었습니다."
+
+#: shared-bindings/bleio/Characteristic.c shared-bindings/bleio/Descriptor.c
+#: shared-bindings/bleio/Service.c
+msgid "Expected a UUID"
+msgstr "UUID이 예상되었습니다."
+
+#: shared-module/_pixelbuf/PixelBuf.c
+#, c-format
+msgid "Expected tuple of length %d, got %d"
+msgstr ""
+
+#: ports/nrf/common-hal/bleio/Device.c
+msgid "Failed to acquire mutex"
+msgstr ""
+
+#: ports/nrf/common-hal/bleio/Characteristic.c ports/nrf/sd_mutex.c
+#, c-format
+msgid "Failed to acquire mutex, err 0x%04x"
+msgstr ""
+
+#: ports/nrf/common-hal/bleio/Service.c
+#, c-format
+msgid "Failed to add characteristic, err 0x%04x"
+msgstr ""
+
+#: ports/nrf/common-hal/bleio/Device.c
+msgid "Failed to add service"
+msgstr "서비스를 추가하지 못했습니다"
+
+#: ports/nrf/common-hal/bleio/Peripheral.c
+#, c-format
+msgid "Failed to add service, err 0x%04x"
+msgstr "서비스를 추가하지 못했습니다., 오류 0x%04x"
+
+#: ports/atmel-samd/common-hal/busio/UART.c ports/nrf/common-hal/busio/UART.c
+msgid "Failed to allocate RX buffer"
+msgstr ""
+
+#: ports/atmel-samd/common-hal/pulseio/PulseIn.c
+#: ports/nrf/common-hal/pulseio/PulseIn.c
+#, c-format
+msgid "Failed to allocate RX buffer of %d bytes"
+msgstr ""
+
+#: ports/nrf/common-hal/bleio/Adapter.c
+msgid "Failed to change softdevice state"
+msgstr ""
+
+#: ports/nrf/common-hal/bleio/Device.c
+msgid "Failed to connect:"
+msgstr 연결할 수 없다""
```

Comment:
Looks like there is a punctuation problem on this line, which may be the reason the build failed.

Reply:
@jepler I fixed it to see what happens now. Feel free to fix the file through github like I did. I think you have permissions too.